ens/utils.py

```diff
@@ -190,7 +191,7 @@ def address_in(address: ChecksumAddress, addresses: Collection[ChecksumAddress])
 
 
 def address_to_reverse_domain(address: ChecksumAddress) -> str:
-    lower_unprefixed_address = remove_0x_prefix(to_normalized_address(address))
+    lower_unprefixed_address = remove_0x_prefix(HexStr(to_normalized_address(address)))
```

Comment:
Not for this PR but it would be ideal if we got our typing right so that we didn't need to wrap this in `HexStr`.